code/game/objects/structures/crates_lockers/closets/secure/locker.dm

```diff
@@ -89,3 +103,25 @@
 		togglelock(usr)
 	else
 		to_chat(usr, SPAN_WARNING("This mob type can't use this verb."))
+
+/obj/structure/closet/secure_closet/verb/reset()
+	set src in oview(1) // One square distance
+	set category = "Object"
+	set name = "Reset Name"
+	if(!usr.canmove || usr.stat || usr.restrained()) // Don't use it if you're not able to! Checks for stuns, ghost and restrain
+		return
+	if(ishuman(usr))
+		add_fingerprint(usr)
+		if (locked || name == initial(name))
+			to_chat(usr, SPAN_WARNING("You need to unlock it first."))
```

Comment:
the message here doesn't make sense if the error case is `name == initial(name)`